server/bin/pbench-index.py

```diff
@@ -428,7 +425,7 @@ def main(options, name):
                         f"Logic Bomb!  Unexpected tar ball handling "
                         f"result status {tb_res:d} for tar ball {tb}"
                     )
-                elif tb_res >= 4 or res <= 11:
```

Comment:
And we might want to add a method to encapsulate what is ignorable.

Also, there appears to be a bug here, s/`res <= 9`/`tb_res < GENERIC_ERROR`.

Reply:
@portante  Sorry, I couldn't understand this "method to encapsulate what is ignorable"

Reply:
If we have a `class` that describes the return codes from indexing tar balls, then we can add a method to that class that will report if a given return value is in the "ignorable" range.